### DISCUSSION TITLE
Media resize image dynamically

### DISCUSSION THREAD

**Discussion by barsdeveloper** [joomla/joomla-cms#34786] [Q&A]
Can Joomla 4 resize images dynamically, based on get params?
I tried with image images/sampledata/cassiopeia/nasa1-1200.jpg
links like:
https://mydomain.com/images/sampledata/cassiopeia/nasa1-1200.jpg?width=200&height=50
or
https://mydomain.com/images/sampledata/cassiopeia/nasa1-1200.jpg#joomlaImage://local-images/sampledata/cassiopeia/nasa1-1200.jpg?width=200&height=50
(this one I get by selecting the offline image)
does not resize the image.

**Comment by brianteeman** [comment]
Simple answer is no.
Longer answer is maybe in the future

**Reply by barsdeveloper** [reply]
Ok, thanks for taking into consideration. Maybe better than arbitrary size that resizes at runtime could be more usefull to create verious version of an image depending on breakpoints defined in global settings. This way the resise would be performed just once when the image is uploaded, the tools to resise are already there.

**Reply by brianteeman** [reply]
See https://volunteers.joomla.org/teams/gsoc-21-media-manager-filter

**Comment by online-today** [comment]
Maybe this is interesting in this context:
https://github.com/joomla-projects/gsoc21_media-manager/pull/16
and now
https://github.com/joomla/joomla-cms/pull/34803 : [4.1] [GSoC 21] Media Manager - Implement Responsive Images with Art
